### PR TITLE
Making sure batchr is in good working order

### PR DESCRIPTION
**Pull request by dunkenwg**

### CODE REVIEW

File: .github/workflows/R-CMD-check.yaml [hunk 1]
```diff
@@ -23,95 +20,33 @@ jobs:
...
-          - {os: macOS-latest,   r: 'release'}
-          - {os: macOS-latest,   r: 'oldrel'}
-          - {os: macOS-11, r: 'release'}
-          - {os: macOS-11, r: 'oldrel'}
+          - {os: macos-latest,   r: 'release'}
           - {os: windows-latest, r: 'release'}
-          - {os: ubuntu-latest, r: 'devel', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
-          - {os: ubuntu-latest, r: 'release', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
-          - {os: ubuntu-latest, r: 'oldrel', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
-          - {os: ubuntu-latest, r: '3.6', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
```
Comment (joethorley) [inline .github/workflows/R-CMD-check.yaml:34]: The DESCRIPTION file still says it works for 3.6 but we are now no longer testing this. The jump from 3.6 to 4.3 seems quite big. Do we have a policy for testing earlier version numbers?

**Review by joethorley — CHANGES_REQUESTED** [review]
Looks great - I have one question

File: .github/workflows/R-CMD-check.yaml (hunk 1 above)
Comment (dunkenwg) [reply .github/workflows/R-CMD-check.yaml:34]: I ran `usepois::use_github_action_check_standard()` (for public packages not on CRAN) which removed the R-CMD-check for several versions of R. Should I have used `usepois::use_github_action_check_full()` even though this package is not yet published on CRAN?

**Review by joethorley — APPROVED** [review]
looks great

**Pull request merged**